hocr-pdf

```diff
@@ -128,12 +128,13 @@ pL26nlWN2K+W1LhRjxlVGKmRTFYVo7CiJug09E+GJb+QocMCPMWBK1wvEOfRFF2U0klK8CppqqvG
 pylRc2Zn+XDQWZIL8iO5KC9S+1RekOex1uOyZGR/w/Hf1lhzqVfFsxE39B/ws7Rm3N3nDrhPuMfc
 w3R/aE28KsfY2J+RPNp+j+KaOoCey4h+Dd48b9O5G0v2K7j0AM6s+5WQ/E0wVoK+pA6/3bup7bJf
 CMGjwvxTsr74/f/F95m3TH9x8o0/TU//N+7/D/ScVcA=
-"""
-  ttf = cStringIO.StringIO(zlib.decompress(base64.decodestring(font)))
+""".encode('latin1')
+  uncompressed = bytearray(zlib.decompress(base64.decodestring(font)))
+  ttf = io.BytesIO(uncompressed)
   pdfmetrics.registerFont(TTFont('invisible', ttf))
 
 if __name__ == "__main__":
   if len(sys.argv) == 1:
-    print("Usage: %s <imgdir>\n" % os.path.basename(sys.argv[0]))
+    print(("Usage: %s <imgdir>\n" % os.path.basename(sys.argv[0])))
```

Comment:
This change seems unrelated to the commit message (and it is a `print` change, which you want to correct at the moment).